eval/eval.py

```diff
@@ -643,47 +784,147 @@ def bprint(s: str):
                 if QUIET:
                     bprint(f"In file {fpath}:")
                 bprint("000: *** Sentence identifier is missing ('n' attribute) ***")
-            gc_error = False
-            ice_error = False
-            # Output GreynirCorrect annotations
-            for ann in s.annotations:
-                if ann.is_error:
-                    gc_error = True
-                if not QUIET:
-                    bprint(f">>> {ann}")
-            # Output iceErrorCorpus annotations
-            xtypes: Dict[str, int] = defaultdict(int)
-            for err in errors:
-                asterisk = "*"
-                xtype = cast(str, err["xtype"])
-                if err["in_scope"]:
-                    # This is an in-scope error
-                    asterisk = ""
-                    ice_error = True
-                    # Count the errors of each xtype
-                    if xtype != "dep":
-                        xtypes[xtype] += 1
+
+
+            def sentence_results(hyp_annotations, ref_annotations):
```

Comment:
Add type annotations

Reply:
Done - tried to do `hyp_annotations: List[Annotation]` but got a NameError, so it's `List[Any]` for now.

Reply:
`List[gc.Annotation]` would probably work. Or, alternatively, add `from reynir_correct import Annotation` at the beginning.

Reply:
Done and pushed!